kratos/python/add_utilities_to_python.cpp

```diff
@@ -424,6 +438,8 @@ void AddUtilitiesToPython(pybind11::module& m)
         .def("CheckDofs", &VariableUtils::CheckDofs)
         .def("UpdateCurrentToInitialConfiguration", &VariableUtils::UpdateCurrentToInitialConfiguration)
         .def("UpdateInitialToCurrentConfiguration", &VariableUtils::UpdateInitialToCurrentConfiguration)
+        .def("UpdateCurrentPosition", VariableUtilsUpdateCurrentPositionNoVariable)
```

Comment:
Better Without

Reply:
I'd leave it as it is. I find it more misleading. At first glance `With` looks similar to `Without`.

Reply:
I find this even more misleading, maybe bettrer nothing at all: VariableUtilsUpdateCurrentPosition

Reply:
OK. I like this one much more.